blog/2024-07-23-july-releases.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: "July 2024 Releases"
+slug: "2024-07-23-july-releases"
+authors: [beeankha]
+tags: [announcement, conda, conda-build, conda-libmamba-solver]
+description: |
+  Conda 24.7.0/.1, conda-build 24.7.0, and conda-libmamba-solver 24.7.0 have been released! 🎉
+image: img/blog/2024-07-23-july-releases/banner.png
+---
+
+The July 2024 releases included updates to conda, conda-build, and conda-libmamba-solver! 🎉 All of these have been released to both `main` and `conda-forge` channels.
+
+## Changes in Conda [24.7.0/.1](https://github.com/conda/conda/releases/tag/24.7.0)
+
+To update conda, run:
+
+```bash
+conda install -n base conda=24.7.1
+```
+
+<!-- truncate -->
+
+### ✨ What's New? ✨
+
+- Add a new `reporters` setting for configuring output.
+- Report traceback of plugin loading errors with verbosity 2 or higher (`-vv` or more).
+- Skip checking for `.pyc` and `.pyo` files in the `conda doctor` "missing files" health check.
+- **Breaking change** `conda list --explicit` will not print authentication details by default. A new flag `--auth` has been added so folks can opt-in to this behavior.
+- Print transaction report for `@EXPLICIT` lockfile installs too.
```

Comment:
Should this have more context? Like what else gets a transaction report?

Reply:
Good call-out. I just removed the "too" since we're just wanting to point out here that the transaction reports now get printed for those lockfile installs.